### DISCUSSION TITLE
Merge conflict

### DISCUSSION THREAD

**Discussion by TSDilov** [dotnet/extensions#4951] [Q&A]
Hi there, I fork and clone the repo on my local machine and after setting an upstream with  'git remote add upstream https://github.com/dotnet/extensions' and 'git pull upstream master' it gives me merge conflicts

**Comment by RussKie** [comment]
This probably means you may have run (or not run) something incorrectly. In either case, this should get you started on a fresh clone:

```
git clone -v --recurse-submodules --progress "https://github.com/TSDilov/extensions.git" "<destination folder>/extensions"
cd ./extensions
git remote add upstream "https://github.com/dotnet/extensions.git"
git fetch --all
```

**Reply by TSDilov** [reply]
Thank you!
